k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientLoader.java

```diff
@@ -341,7 +343,7 @@ private void fillCryptoStatusData(Map<String, Recipient> recipientMap) {
         String[] recipientAddresses = recipientList.toArray(new String[recipientList.size()]);
 
         Cursor cursor;
-        Uri queryUri = Uri.parse("content://" + cryptoProvider + ".provider.exported/email_status");
+        Uri queryUri = Uri.parse("content://" + cryptoProvider + ".provider.exported/autocrypt_status");
```

Comment:
What is happening if this is used with an old OpenKeychain version? Does the app silently fail?

Reply:
that should never happen, see 5688619d9f54f71b7546d1b482cf64137a36d116